Amsterdam/signals-frontend
# [feat-5457]: Add packages and initialize i18next with default json

Ticket: [SIG-5457](https://gemeente-amsterdam.atlassian.net/browse/SIG-5457)

### Context
This PR introduces i18next with a default json which serves as fallback when the backend doesn't return translations

note:
- the labels under `algemeen` are translated from the language setting of the app.base.json file because we want Dutch namespaces here. 

### Changes
- Add i18next and related packages
- Initialize i18next. 
- Fetch json from backend
- Add default json as fallback

### Testing
- backend test file serves a `test` key
- Add `test` key to the fall back json ("key":  "this is local fallback key)
- Implement this key somewhere using `  const { t } = useTranslation()`
- See the backend value appear
- when disabling this call, the fall back should appear

[SIG-5457]: https://gemeente-amsterdam.atlassian.net/browse/SIG-5457?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
